server/budgettracker.data/BudgetTrackerContext.cs

```diff
@@ -6,10 +7,17 @@ namespace budgettracker.data
 {
     public class BudgetTrackerContext : IdentityDbContext<UserModel>
     {
+        public BudgetTrackerContext()
```

Comment:
Why does this need to be here? If you instantiate it without options, then it doesn't know how to connect to the DB

Reply:
What are the options that I need to passs in?